src/components/z-dragdrop-area/styles.css

```diff
@@ -0,0 +1,56 @@
+:host {
+  font-family: var(--font-family-sans);
+  font-weight: var(--font-rg);
+}
+
+:host > .dragdrop {
+  display: flex;
+  color: var(--gray800);
+  flex-direction: column;
+  position: relative;
+  /* Native CSS properties don't support the customization of border-style. 
+  Therefore, we use a trick with an SVG image inside background-image property. 
+  The SVG features give us the ability to change the distance between dashed lines, set custom pattern, add dash offset or even change a line cap. 
+  Generated SVG image is vector and it fills width and height of elements by 100%, so it doesn't matter what size elements have. */
+  background-image: url("data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' rx='2' ry='2' stroke='%23CACCCEFF' stroke-width='2' stroke-dasharray='15%2c 10%2c 14%2c 11' stroke-dashoffset='3' stroke-linecap='square'/%3e%3c/svg%3e");
+  border-radius: 2px;
```

Comment:
Mancano un po' di variabili CSS in tutto il file

Reply:
sì perchè ci sono delle cose che non mi tornano e volevo sentire Daniele, intanto le ho messo così.